models/nlp/classification/huggingface/requirements.txt

```diff
@@ -0,0 +1,8 @@
+transformers
+datasets
+torch==1.13.1
+scikit-learn
+evaluate
+valohai-utils==0.3.1
+accelerate
+nvidia-ml-py3
```

Comment:
Repeated for all of the requirements files:

* Maybe loosen the pin for valohai-utils to e.g. `valohai-utils~=0.3.1`? That'd allow everything in the 0.3 series that's equal to or newer than 0.3.1. Or `~=0.3` to allow everything >=0.3,<1.
* I don't see `accelerate`, `scikit-learn` or `evaluate` directly referenced by anything in this PR – are they necessary for some hidden reason?